tests/reducers/activeOrderReducer.test.js

```diff
@@ -146,22 +146,6 @@ describe('Active Order Reducer', () => {
     expect(newState.activeOrders[0].status).toEqual(action.action);
   });
 
-  it('should set the DISCARD_EDIT order action', () => {
```

Comment:
You need to update this PR with the merged changes

Reply:
Done